### ISSUE TITLE
Worldguard flag doesnt exist

### ISSUE THREAD

**Issue by markusew** [snowgears/shopbugs#172]
**Describe the bug**
Hey, I reached out on the spigot discussion thread but I really can't get it to work and I'm assuming its a bug. So it says the flag doesnt exist, and I can't find an available flag that could be related to the shop plugin. This is the list of flags I have available (with shop installed and worldguard enabled in the config. But its worth nothing that I can't create shops with it enabled cause of the lacking flag in the region)
```(FAWE) Unknown flag specified: allow-shop
Available flags: allow-block-break, allow-block-place, allowed-cmds, block-break, block-place, block-trampling, blocked-cmds, blocked-effects, build, chat-prefix, chat-suffix, chest-access, chorus-fruit-teleport, chunk-unload, command-on-entry, command-on-exit, console-command-on-entry, console-command-on-exit, coral-fade, creeper-explosion, crop-growth, damage-animals, deny-block-break, deny-block-place, deny-message, deny-spawn, enderdragon-block-damage, enderman-grief, enderpearl, entity-item-frame-destroy, entity-painting-destroy, entry, entry-deny-message, exit, exit-deny-message, exit-override, exit-via-teleport, exp-drops, fall-damage, farewell, farewell-title, feed-amount, feed-delay, feed-max-hunger, feed-min-hunger, fire-spread, firework-damage, fly, frosted-ice-form, frosted-ice-melt, frostwalker, game-mode, ghast-fireball, give-effects, glide, godmode, grass-growth, greeting, greeting-title, heal-amount, heal-delay, heal-max-health, heal-min-health, ice-form, ice-melt, interact, invincible, item-drop, item-durability, item-frame-rotation, item-pickup, join-location, keep-exp, keep-inventory, lasso, lava-fire, lava-flow, lay, leaf-decay, lighter, lightning, mcmmo, mcmmo-hardcore, mcmmo-xp, mob-damage, mob-spawning, mushroom-growth, mycelium-spread, mythicmobs-eggs, natural-health-regen, natural-hunger-drain, nether-portals, notify-enter, notify-leave, other-explosion, passthrough, pistons, play-sounds, player-particles, potion-splash, pvp, ravager-grief, receive-chat, respawn-location, ride, send-chat, sit, sleep, snow-fall, snow-melt, snowman-trails, soil-dry, spawn, teleport, teleport-message, teleport-on-entry, teleport-on-exit, time-lock, tnt, use, vehicle-destroy, vehicle-place, vine-growth, walk-speed, water-flow, weather-lock, wither-damage, worldedit, Or use the command /rg flags __global__
```

**To Reproduce**
Try setting the flag allow-shop in a region. It will say the flag doesnt exist.

**Version:**
 - Spigot version: [Paper 1.16.3 (Tuinity)]
 - Shop version: [1.8.2.3]

**Comment by snowgears** [comment]
What other plugins are you using? This sounds like another plugin incorrectly registering the same flag.

**Comment by markusew** [comment]
I was using a bunch, but switching over to these 7 only and the issue is still there
```Essentials, EssentialsChat, EssentialsSpawn, (WorldEdit)FastAsyncWorldEdit, Shop, Vault, WorldGuard```

**Comment by snowgears** [comment]
Sounds like this may be due to registering a flag as a non OP player. Reported by TatoIQ in the discord:

`Way to produce:
Case 1:
1- Let none OP player place chest  inside a GriefPrevention claim
2- Player places a sign inside a GP claim, and tries to make a shop, it doesn't covert to the shop sign and gives the error
* If the player is OP there would be no error.

Case 2:
1- If OP or NONE player tried to make a shop inside a WorldGaurd region where there is no GP region, there will be no error and  it will covert the shop sign correctly

Case 3:
1- If OP or NONE player tried to make a shop  but outside WG and GP regions like inside no regions, there will be no error and  it will covert the shop sign correctly

Case 4:
1- If OP or NONE player tried to make a shop inside a region where WG and GP regions overlap, there would be no errors either.`

**Comment by untuned** [comment]
I am also getting this @snowgears, the Shop plugin does not appear to be registering the flag.

**Comment by snowgears** [comment]
Fixed in 1.8.2.5 (in progress)
